cognite/neat/_graph/loaders/_rdf2dms.py

```diff
@@ -231,6 +232,29 @@ def sort_by_instance_type(id_: dm.ViewId) -> int:
             view_iterations.append(view_iteration)
         return view_iterations, issues
 
+    @staticmethod
+    def _lookup_max_limits_size(client: NeatClient, views: dm.ViewList) -> None:
+        """For listable container properties (mapped properties), the read version of the view does not
+        contain the max_list_size. This method will lookup the max_list_size from the containers definitions."""
+        containers = client.data_modeling.containers.retrieve(list(views.referenced_containers()))
+        properties_by_container_and_prop_id = {
+            (container.as_id(), prop_id): prop
+            for container in containers
+            for prop_id, prop in container.properties.items()
+        }
```

Comment:
This is necessary due to maxListSize is missing in response/read format of view:
![image](https://github.com/user-attachments/assets/d9b341e2-c3e5-464e-bca1-ada2757b360e)

This is reported to the API team, but for now, this is needed to migration.